src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -196,7 +175,7 @@ export class MeetingManager implements AudioVideoObserver {
   initializeMeetingManager(): void {
     this.meetingSession = null;
     this.audioVideo = null;
-    this.configuration = null;
+    this.meetingSessionConfiguration = null;
```

Comment:
Prefer using `undefined` instead of `null`.

Reply:
Done